.drone.yml

```diff
@@ -24,7 +24,43 @@ steps:
     commands:
       - npm run test
 
-  - name: build
+  - name: build staging
+    image: node:12
+    pull: always
+    environment:
+      DIRECTUS_URL:
+        from_secret: directus_url
+      DIRECTUS_TOKEN:
+        from_secret: directus_token
+    commands:
+      - npm run export:staging
```

Comment:
I thing it might be cleaner to use the same build command as prod, just setting the right env variable above `NODE_ENV=staging`